app/src/main/java/com/example/musicapp/service/InstrumentService.kt

```diff
@@ -11,31 +11,26 @@ import android.util.Log
 
 class InstrumentService {
 
-    fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
+    internal fun fetchInstruments(): MutableLiveData<ArrayList<Instrument>> {
         var _instruments = MutableLiveData<ArrayList<Instrument>>()
         val service = RetrofitClientInstance.retrofitInstance?.create(MusicAppDAO::class.java)
         val call = service?.getAllInstruments()
-        call?.enqueue(object: Callback<ArrayList<Instrument>>{
+        call?.enqueue(object : Callback<ArrayList<Instrument>> {
```

Comment:
This is another suggestion that is in line with my Code Review Suggestion 1. This time, there should be a space before the colon since according to https://kotlinlang.org/docs/reference/coding-conventions.html, a space should always be before the colon following the object keyword.